### PR TITLE
Added Jest and parse-docket-pdf.utils test (#99)

### PR DESCRIPTION
**Pull request by HappyViki**
Added Jest
Added obfuscated file in correct format
Added a bunch of tests for parse-docket-pdf.utils and that obfuscated file

### CODE REVIEW

File: backend/apis/parse-docket-pdf.utils.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const { file } = require("./docket-pdf-test-files/file");
+const { parsePdfText } = require("./parse-docket-pdf.utils");
```
Comment (joeldenning) [inline backend/apis/parse-docket-pdf.utils.test.js]: So I think that `parse-docket-pdf.utils.test.js` should maybe be renamed to `normal-docket-pdf.test.js`. The idea is that we are going to have many different docket pdfs that have their own tests, each one verifying different edge cases with the PDFs. So we want the file names to not be as generic.

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "build": "yarn webpack",
     "analyze": "webpack --config webpack.config.analyze.js",
     "check-format": "prettier 'frontend/**/*' 'backend/**/*' --check",
-    "deploy": "yarn build && gh-pages -d dist"
+    "deploy": "yarn build && gh-pages -d dist",
+    "test": "jest"
```
[the remark this answers is not in the document]
Comment (joeldenning) [reply package.json]: btw have you tried out `yarn test --watch`?? It makes developing the tests a lot easier!

File: backend/apis/parse-docket-pdf.utils.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ function parseCharges(lines, sections) {
       });
     } else if (line.startsWith("Disposition:")) {
       const numeric = /\d/;
-      lastNumericIndex = 0;
+      let lastNumericIndex = 0;
```
Comment (joeldenning) [inline backend/apis/parse-docket-pdf.utils.js]: 👍 good catch

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (HappyViki) [reply package.json]: I think I have before. But forgot. Thanks for reminding me!

File: backend/apis/parse-docket-pdf.utils.test.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+const parsedText = parsePdfText(file);
+
+//Bad:
```
[the remark this answers is not in the document]
Comment (HappyViki) [reply backend/apis/parse-docket-pdf.utils.test.js]: I was using it for reference, but will delete, because yes, it's not needed.

File: backend/apis/parse-docket-pdf.utils.js (hunk 3 above)
Comment (HappyViki) [reply backend/apis/parse-docket-pdf.utils.js]: For future reference, I knew how to fix it because I got this error:
`ReferenceError: lastNumericIndex is not defined`
